libvmi/driver/kvm/kvm.c

```diff
@@ -76,33 +76,46 @@ exec_qmp_cmd(
     char *query)
 {
     FILE *p;
-    char *output = safe_malloc(20000);
+    char *output = g_malloc0(20000);
+    if ( !output )
+        return NULL;
```

Comment:
I don't think we have a hard-rule that requires it either way. Since all the rest of the code here returns NULL instead of goto exit, this is IMHO also fine.